themes/finna2/templates/RecordDriver/DefaultRecord/holdings.phtml

```diff
@@ -1,19 +1,24 @@
 <!-- START of: finna - RecordDriver/DefaultRecord/holdings.phtml -->
+
 <?php
   $dedupData = $this->driver->getDedupData();
   $deduplicated = count($dedupData) > 1;
+  $selectID = "dedup-select-" . Finna\View\Helper\Root\Record::$dedupCounter++;
```

Comment:
Tämä on vähän häkkyrä. Ennemmin vaikka helpperi tms., jolta saa juoksevan numeron tyyliin `$selectID = 'dedup-select-' . $this->counter()->increment()`. Silloin ei tarvitse edes mitään static:ia, kun helpperi on aina sama instanssi.